homeassistant/components/ituran/strings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id_or_passport": "ID or passport number",
+          "phone_number": "Mobile phone number"
+        },
+        "data_description": {
+          "id_or_passport": "The goverment ID or passport number provided when registering with Ituran.",
```

Comment:
```suggestion
          "id_or_passport": "The government ID or passport number provided when registering with Ituran.",
```

Reply:
Oh whoops

Reply:
I'll fix this in a follow-up commit. Thanks!